Windows에서의 UnicodeEncodeError

## 환경
- Windows
## 내용
[utils/datetime.py](https://github.com/team-crescendo/Crenata/blob/main/crenata/utils/datetime.py#L41)에서 UnicodeEncodeError가 발생합니다
## 재현
Windows 환경에서 datetime.strftime()의 매개변수에 한글을 포함하는 문자열 작성

모바일에서 작성해 가독성에 문제가 있을 수 있습니다

운영체제: Windows 11 Pro (22H2)
Python: 3.10.10
재현 명령어:
```pwsh
poetry run pytest --asyncio-mode=auto
```
오류:
```py
date = datetime.datetime(2021, 1, 1, 0, 0)

    def datetime_to_readable(date: datetime) -> str:
        """
        datetime을 YYYY년 MM월 DD일 형식의 문자열로 변환합니다.
        """
>       return date.strftime("%Y년 %m월 %d일")
E       UnicodeEncodeError: 'locale' codec can't encode character '\ub144' in position 2: encoding error

crenata\utils\datetime.py:41: UnicodeEncodeError
```

--- a/crenata/utils/datetime.py
+++ b/crenata/utils/datetime.py
@@ -38,7 +38,7 @@ def datetime_to_readable(date: datetime) -> str:
     """
     datetime을 YYYY년 MM월 DD일 형식의 문자열로 변환합니다.
     """
-    return date.strftime("%Y년 %m월 %d일")
+    return date.strftime("%Y{} %m{} %d{}").format("년", "월", "일")
 
 
 def to_weekday(date: datetime) -> str:

